### ISSUE TITLE
If the value of SecRuleEngine is DetectionOnly, where should I see the access that matches the rule but is not blocked

### ISSUE THREAD

**Issue by choufenging** [coreruleset/coreruleset#3084]

**Comment by airween** [comment]
hi @choufenging,

> But I didn't actually see the warning in nginx's error log.

In case of Nginx you should add the log level to your config, eg:

```
    error_log /var/log/nginx/error.log info;
```

> Can I only see the warning content in /var/log/modsec_audit.log?

Yes, but it depends on you `SecAuditLogParts` setting. Please see the [documentation](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-(v3.x)#SecAuditLogParts).

Why did you add the label "False Positive"?

**Comment by choufenging** [comment]
Thank you very much, I'll take a look.
@airween 
The reason I'm posting this on this label is that I'm not quite sure which label to add it on
